application/templates/alertmanagerconfig.yaml

```diff
@@ -6,15 +6,15 @@ metadata:
   name: {{ template "application.name" . }}
   namespace: {{ include "application.namespace" . }}
   labels:
-  {{- include "application.labels" . | nindent 4 }}
+  {{- include "application.labels" $ | nindent 4 }}
```

Comment:
Can you explain what $ does here that wasn't already covered by `.` ?

Reply:
This change and some other similar ones can be reverted. I mass replaced include statements. Using $ there ensures that even if they are wrapped in a range in the future, they get th le right context.